queue/amqp.go

```diff
@@ -72,27 +72,43 @@ func connect(url string) (*amqp.Connection, *amqp.Channel) {
 	// first try to connect again
 	var conn *amqp.Connection
 	var err error
+	var lastErr error
 	for {
 		conn, err = amqp.Dial(url)
 		if err != nil {
-			log15.Error("error connecting to amqp", "err", err)
+			if lastErr == nil || err.Error() != lastErr.Error() {
+				log15.Error("error connecting to amqp", "err", err)
+				log15.Debug("retrying connect to amqp...")
```

Comment:
Here the log level is `debug` but for amqp channel creation is `info`. I would have them with the same log level.